src/components/Plugins/NodeInfo/NodeInfoDot.js

```diff
@@ -115,7 +116,14 @@ class NodeInfoDot extends Component {
   }
 }
 
-export default NodeInfoDot
+function mapStateToProps(state) {
+  return {
+    blockNumber:
+      Number(state.plugin[state.plugin.selected].active.blockNumber) || 0
+  }
+}
+
+export default connect(mapStateToProps)(NodeInfoDot)
```

Comment:
Is Redux necessary here? `blockNumber` is already being used elsewhere in this component. Would prefer to keep this component unconnected unless really necessary.

Reply:
good point, fixed